src/routes/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { RequestEvent } from '@sveltejs/kit';
+
+import { getLocale } from '$lib/utilities/i18n.utilities';
+
+// Supported locales
+import ca_en from '$locales/ca/en/index.json';
+import ca_fr from '$locales/ca/fr/index.json';
+import us_en from '$locales/us/en/index.json';
+import us_es from '$locales/us/es/index.json';
+
+const locales = {
+  ca_en,
+  ca_fr,
+  us_en,
+  us_es,
+}
+
+export async function get(event: RequestEvent) {
+  const userLocale = event.request.headers.get('accept-language') || 'en-US';
+
+  const locale = getLocale(userLocale, '_');
+
+  return {
+    body: {
+      content: locales[locale],
```

Comment:
This gets mapped to the `export let content` in the `index.svelte` page component.